archiver_slackr.rb

```diff
@@ -134,16 +134,25 @@
   channels_to_archive.each do |channel|
     channel_id = channel[:channel_id]
     channel_name = channel[:channel_name]
-    client.channels_archive(channel: channel_id)
-    client.chat_postMessage(
-      channel: channel_id,
-      text: "Hello! This channel has been found ded and has been archived. Please contact IT with any memoriam or resurrection requests."
-      )
-    archiver_log.info { "[#{channel_name}] has been archived!" }
-    # add it to the historical log of archived_channels
-    File.open("slackr_archived_channels.log", "a") { |log| log.write("#{channel_name} was archived at #{run_date}\n") }
-    # sleep for slack rate limiting
-    sleep 1
+    is_archived = client.channels_info(channel: channel_id)
+    is_archived = is_archived.channel.is_archived
+
+    if !is_archived  # if channel is not archived let's do stuff
```

Comment:
Can you please move the comment from beside the `if` to above the `if`? Like so:

```ruby
# If channel is not archived let's do stuff
if !is_archived
```